add infrahub (#157)
networktocode/awesome-network-automation: README.md

 - [Using Chef Client with Cisco NX-OS](https://www.cisco.com/c/en/us/td/docs/switches/datacenter/nexus9000/sw/7-x/programmability/guide/b_Cisco_Nexus_9000_Series_NX-OS_Programmability_Guide_7x/b_Cisco_Nexus_9000_Series_NX-OS_Programmability_Guide_7x_chapter_01110.html) - Using Chef Client with Cisco NX-OS

## Infrahub

> - [Infrahub GitHub Repo](https://github.com/opsmill/infrahub) - Infrahub offers a central hub to manage the data, templates and playbooks that powers your infrastructure by combining the version control and branch management capabilities of Git with the flexible data model and UI of a graph database.
> - Getting Started with Infrahub
>   - [Infrahub Documentation](https://docs.infrahub.app/) - Including overview, getting started and FAQ.

Reviewer: ```suggestion
- [Infrahub GitHub Repo](https://github.com/opsmill/infrahub) - Infrahub - A new approach to Infrastructure Management.
- [Infrahub Documentation](https://docs.infrahub.app/) - Including overview, getting started, and FAQ.
```

 - [Using Chef Client with Cisco NX-OS](https://www.cisco.com/c/en/us/td/docs/switches/datacenter/nexus9000/sw/7-x/programmability/guide/b_Cisco_Nexus_9000_Series_NX-OS_Programmability_Guide_7x/b_Cisco_Nexus_9000_Series_NX-OS_Programmability_Guide_7x_chapter_01110.html) - Using Chef Client with Cisco NX-OS

## Infrahub

> - [Infrahub GitHub Repo](https://github.com/opsmill/infrahub) - Infrahub offers a central hub to manage the data, templates and playbooks that powers your infrastructure by combining the version control and branch management capabilities of Git with the flexible data model and UI of a graph database.

Reviewer: This is generally a copy/paste from the "about" on github, which currently is "Infrahub - A new approach to Infrastructure Management", can you have these two match?

Author: I was trying to keep the contents in this description short and to the point, inline with the amount of text on all other listings in this README, where the "about" section on github can be more verbose. "A new approach" doesn't actually say what the project does. So not quite sure how to reconcile this?

> - Getting Started with Infrahub

Reviewer: Let's just do the docs page here, there should be a full sentence ending with a period as a description.

Author: Simplified, let me know if this is better!
